lib/woql.js

```diff
@@ -92,17 +92,62 @@ WOQL.and = function(...subqueries) {
 
 
 /**
- * Read a node identified by an IRI as a JSON-LD document
+ * [Instead use read_document] Read a node identified by an IRI as a JSON-LD document
```

Comment:
Rather than add this, can you [add a message to `@deprecated`](https://jsdoc.app/tags-deprecated.html)?

Reply:
I have added it but it was not adding the text after the @deprecated so i added the tag and wrote the line here

Reply:
I see! Fine then.

Reply:
I learned from @Francesca-Bit that you can [`@link`](https://jsdoc.app/tags-inline-link.html) another name. So I suggest removing all documentation for `read_object` and adding something like this:

```
/**
 * Use {@link WOQL.read_document} instead.
 * @deprecated
 */
```

Just check the link to make sure it's correct. I haven't tested it.